### PR TITLE
Go proxy

### PR DESCRIPTION
**Pull request by aranhoide**

### CODE REVIEW

File: salt/fallback_proxy/install_http_proxy.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env sh
+
+echo 'Installing golang 1.5.1 ...'
+curl -LO https://github.com/getlantern/http-proxy/releases/download/v0.0.1/http-proxy
```
Comment (myleshorton) [inline salt/fallback_proxy/install_http_proxy.sh]: This doesn't actually install golang right? Is that done somewhere else @aranhoide?

Comment (aranhoide) [inline salt/fallback_proxy/install_http_proxy.sh]: Heh, I saw that printout but forgot to fix it.  That downloads the http-proxy binary.  We aren't installing golang at all currently.

Comment (myleshorton) [inline salt/fallback_proxy/install_http_proxy.sh]: Oh right -- I guess we don't need go with the binary itself! Makes sense @aranhoide.

Comment (fffw) [inline salt/fallback_proxy/install_http_proxy.sh]: Blame me please. I made that confusion.

**Pull request merged**